dart_test.yaml

```diff
@@ -6,8 +6,4 @@ paths:
 
 concurrency: 4
```

Comment:
doesn't really hurt anything, but unless tests need a concurrency of 1, the default for this usually works just fine - iirc, it defaults to half the number of available cores